### ISSUE TITLE
No lint for Definition Lists

### ISSUE THREAD

**Issue by emeraldinspirations** [DavidAnson/vscode-markdownlint#315]
## When user

- Has `markdownlint` v0.53.0 installed in the Visual Studio Code editor with the default settings
- Creates markdown file
- Adds the following code:

```
Definition Title
: Definition Data

Definition Title 2
: Definition Data 2
```

- Saves the file

## What user expects

- Line breaks to be preserved due to Definition List syntax

## What actually happens

- File is saved as:

```
Definition Title : Definition Data

Definition Title 2 : Definition Data 2
```

## Notes

- Definition lists are part of the extended syntax 
  <https://www.markdownguide.org/extended-syntax/#definition-lists>
- Suggestion: Add an optional lint to preserve the definition list extended syntax

**Comment by DavidAnson** [comment]
This extension does not do anything special for definition lists as they are not part of the CommonMark/GFM specifications. The text you show above does not produce any violations from this tool, so it should not be suggesting any changes. By default, this extension does not make changes automatically. For these reasons, I think the behavior you report is coming from a different extension, perhaps prettier. You can confirm this by temporarily disabling or uninstalling the markdownlint extension and trying to reproduce the issue.

https://dlaa.me/markdownlint/#%25m%23%20Issue%20315%0A%0ADefinition%20Title%0A%3A%20Definition%20Data%0A%0ADefinition%20Title%202%0A%3A%20Definition%20Data%202%0A

**Comment by emeraldinspirations** [comment]
Apologies, thank you for the clarification @DavidAnson
